main.js

```diff
@@ -101,6 +101,12 @@ module.exports = function(options) {
 		});
 	}
 
+	if (require(path.join(process.cwd(), 'package.json')).dependencies['next-metrics']) {
```

Comment:
oh actually if you're using `require` it uses `path.join` internally so it's not necessary